components/PredictPage/WhatIsOlasPredict.jsx

```diff
@@ -0,0 +1,218 @@
+import Image from 'next/image';
+import {
+  BicepsFlexed, Expand, HandCoins, Sparkles,
+} from 'lucide-react';
+
+import {
+  SCREEN_WIDTH_LG,
+  SUB_HEADER_CLASS,
+  SUB_HEADER_MEDIUM_CLASS,
+  TEXT_MEDIUM_CLASS,
+  TEXT_CLASS,
+} from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+
+const agentRolesList = [
+  {
+    img: 'market-creators',
+    title: 'Market Creators',
+    desc: (
+      <>
+        Deploy and seed prediction markets using the prediction market
+        protocols, specifically
+        {' '}
+        <a
+          href="https://aiomen.eth.limo/"
```

Comment:
not really blocking but this link seems broken for me, just infinity loads, have tried to access this market a few times, any idea why it doesn't load @oaksprout?